include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -248,6 +248,8 @@ struct SessionOptions : Base<OrtSessionOptions> {
   SessionOptions& DisablePerSessionThreads();
 
   SessionOptions& AddConfigEntry(const char* config_key, const char* config_value);
+
+  SessionOptions& AddInitializer(const char* name, OrtValue* ort_val);
```

Comment:
>OrtValue [](start = 51, length = 8)

Question - we can't allow it be a const right? Since deep inside somewhere ort_val will have to be updated?